bi-connectors/PowerBIConnector/OpenSearchProject-ODBC.pq

```diff
@@ -1,18 +1,18 @@
 // This file contains Data Connector logic
 [Version = "1.0.0"]
-section OpenSearch;
+section #"OpenSearchProject-ODBC";
```

Comment:
Remove -ODBC

Reply:
This should be "OpenSearchProject" not just "OpenSearch"

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008